ENIGMAsystem/SHELL/Platforms/SDL/Gamepad.h

```diff
@@ -0,0 +1,55 @@
+#ifndef GAMEPAD_H
+#define GAMEPAD_H
+
+#include <vector>
+
+namespace enigma_user {
+enum {
+
+  gp_any = -1,
```

Comment:
Are these SDL-specific, or should they go under General?

Reply:
Xinput extension uses them as well but Linux joystick doesn't